src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -261,10 +260,6 @@ public function configureOptions(OptionsResolver $resolver)
             if ($choiceList) {
                 @trigger_error('The "choice_list" option is deprecated since version 2.7 and will be removed in 3.0. Use "choice_loader" instead.', E_USER_DEPRECATED);
```

Comment:
hmm, removing the option entirely may require an additional change in 2.8 first, to have code compatible with both ways to build the choice list. It probably deserve to be done separately.